root/report/VideosInNonVideoMediums.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import RecordingList from './components/RecordingList';
+import ReportLayout from './components/ReportLayout';
+import type {ReportDataT, ReportRecordingT} from './types';
+
+const VideosInNonVideoMediums = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportRecordingT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows recordings marked as video, but that appear in
+       at least one medium that does not support videos.`,
+    )}
+    entityType="recording"
+    extraInfo={exp.l(
+      `There are two main possibilities here: either the recording being
+       marked as a video is correct, but the format is not (a CD should be a
+       VCD, for example), or the recording is being used for both a video and 
+       and audio-only recording, in which case the two should be split since
+       video recordings should always be separate.`,
+    )}
```

Comment:
Extra guidance if the two recordings should always be linked through a music video relationship:

```suggestion
    extraInfo={exp.l(
      `There are two main possibilities here: either the recording being
       marked as a video is correct, but the format is not (a CD should be a
       VCD, for example), or the recording is being used for both a video and 
       and audio-only recording, in which case the two should be split since
       video recordings should always be separate and linked through a
       {doc_link|music video relationship}.`,
      {doc_link: '/relationship/ce3de655-7451-44d1-9224-87eb948c205d'},
    )}
```

Reply:
Only if it's a video for the same audio, I guess :) It's probably often the case, but not always - I will mention the relationship, anyway, that's a good point.

Reply:
Your [changes](https://github.com/metabrainz/musicbrainz-server/compare/d2e84139ed368bb0759488d35f23edbd163f5ecc..bf30f276b118774508b73179bbf4544a9c6c28a9) look good, thanks!